src/facades/book.js

```diff
@@ -2,6 +2,16 @@
 
 let books = require('../../db.json')
 
+function getAuthors () {
+  let result = new Set(
+    books
+      .map(book => book.authors)
+      .reduce((prev, curr) => prev.concat(curr))
+      .sort()
+    )
+  return [...result]
```

Comment:
I like how you used new ES6 features like `Set` datatype with  `map() + reduce()` a good example of a functional approach as well as a _destructuring_ with `[...]` 👍